WcaOnRails/app/models/competition_venue.rb

```diff
@@ -46,6 +46,7 @@ def to_wcif
       "name" => name,
       "latitudeMicrodegrees" => latitude_microdegrees,
       "longitudeMicrodegrees" => longitude_microdegrees,
+      "countryIso2" => country_iso2,
```

Comment:
@jonatanklosko, FYI I did not backfill this data for existing competitions, so it will be empty for all competitions. We could do a backfill, but we should probably have a human look at every competition with multiple venues.

Reply:
Having a quick look into the database it looks like we have only 9 such competitions (7 visible) and all of them have only 2 venues, so most likely are in the same country anyway =)

Reply:
Here's a list of all competitions in the database with multiple `competition_venues`:

```sql
mysql> SELECT CONCAT("https://www.worldcubeassociation.org/competitions/", c.id, "/schedule/edit") FROM Competitions c JOIN competition_venues cv ON cv.competition_id = c.id GROUP BY c.id HAVING COUNT(*) > 1;
+-------------------------------------------------------------------------------------------+
| CONCAT("https://www.worldcubeassociation.org/competitions/", c.id, "/schedule/edit")      |
+-------------------------------------------------------------------------------------------+
| https://www.worldcubeassociation.org/competitions/CubingSemeyOpen2019/schedule/edit       |
| https://www.worldcubeassociation.org/competitions/denemeGizem2019/schedule/edit           |
| https://www.worldcubeassociation.org/competitions/IdealeNabeulOpen2019/schedule/edit      |
| https://www.worldcubeassociation.org/competitions/KSFAtyrauCity2019/schedule/edit         |
| https://www.worldcubeassociation.org/competitions/MedanFarewell2019/schedule/edit         |
| https://www.worldcubeassociation.org/competitions/MindanaoChampionships2018/schedule/edit |
| https://www.worldcubeassociation.org/competitions/PolishChampionship2019/schedule/edit    |
| https://www.worldcubeassociation.org/competitions/SucursalDelCielo2019/schedule/edit      |
| https://www.worldcubeassociation.org/competitions/SwisscubingFMC2018/schedule/edit        |
+-------------------------------------------------------------------------------------------+
9 rows in set (0.01 sec)
```

Reply:
I just checked all these competitions, and they all happen in 1 country, so I think we can do a correct backfill here. I will update my migration accordingly.

Reply:
Ok, backfill added to the migration!